content/momentum/changelogs/M4ChangeLog.md

```diff
@@ -3,6 +3,8 @@
 
```

Comment:
The recent Momentum on prem doc updates have been done in the support-docs repo and that repo also has changelogs (https://github.com/SparkPost/support-docs/tree/master/momentum/changelog/4). 

Why do we have both support-docs/tree/master/momentum/ and developers.sparkpost.com/content/momentum/?

I feel like we've had this discussion before but I don't remember the outcome.  I'm hoping we can kill one of these directories.